### PR TITLE
fix the "Basic cookbooks" link

### PR DESCRIPTION
**Pull request by vmsilvamolina**
## PR Summary

I found a mistake on the PowerShell Web Docs: Basic cookbooks link, but searching on the documentation, the only link that mention "cookbook", is the suggested. 

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [x] Not Applicable or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

**Comment by vmsilvamolina** [comment]
It's right? Is any file missing?

File: docs/learning-powershell/README.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ Recommended Training and Reading
 ----
 - Microsoft Virtual Academy: [Getting Started with PowerShell][getstarted-with-powershell]
 - [Why Learn PowerShell][why-learn-powershell] by Ed Wilson
-- PowerShell Web Docs: [Basic cookbooks][basic-cookbooks]
+- PowerShell Web Docs: [Basic cookbooks][https://docs.microsoft.com/en-us/powershell/scripting/getting-started/fundamental/sample-scripts-for-administration?view=powershell-6]
```
Comment (iSazonov) [inline docs/learning-powershell/README.md]: Please use the file pattern and move the url to eof (add `[basic-cookbooks]:url`).

**Review by iSazonov — APPROVED** [review]

**Comment by iSazonov** [comment]
Appveyor CI temporary failed.

**Comment by vmsilvamolina** [comment]
@iSazonov how can I help to solve the AppVeyor error?

**Comment by SteveL-MSFT** [comment]
@vmsilvamolina I restarted build on AppVeyor as it's not related to your changes

**Review by SteveL-MSFT — APPROVED** [review]
LGTM

**Comment by TravisEz13** [comment]
I had to restart AppVeyor again.  Ping me if I don't update the PR within a day.

**Comment by anmenaga** [comment]
@TravisEz13 This looks ready for merge.

**Comment by iSazonov** [comment]
@vmsilvamolina Thanks for your contribution!

**Pull request merged**